src/constants.js

```diff
@@ -12,4 +12,6 @@ export default {
     MY_PUBLIC_APPS: "My public apps",
     HPC: "High-Performance Computing",
     BROWSE_ALL_APPS: "Browse All Apps",
+    EXTERNAL_APP_TYPE: "External",
```

Comment:
In my App Launch branch, I added this constant to a `src/components/apps/constants.js` file, since I thought this probably won't need to be referenced outside of apps components (I could be wrong).

I also named it `APP_TYPE_EXTERNAL`, in case we need any more app types, then they will be easier to read/scan:
APP_TYPE_A
APP_TYPE_B
APP_TYPE_C
etc.

Reply:
I thought we might need `AppTypes` filtering in `Analyses`.